switchmap/server/db/ingest/ingest.py

```diff
@@ -470,11 +470,10 @@ def insert_arptable(data, test=False):
     )
     log.log2debug(1084, log_message)
     if bool(test) is False:
-        _mac.insert_row(macs)
+        _mac.bulk_insert_rows(macs)
```

Comment:
1. Revert.
2. Update `db.add_all` to use the new `bulk_insert_rows` method
3. Apply this to all the` switchmap/server/db/table` files. It's simpler. There v2.0 of sqlalchemy has a `bulk_insert_rows` replacement.